server/svix-server/src/core/cache/redis.rs

```diff
@@ -116,11 +116,16 @@ mod tests {
     async fn get_pool(redis_dsn: &str, cfg: &crate::cfg::Configuration) -> RedisPool {
         match cfg.cache_type {
             CacheType::RedisCluster => crate::redis::new_redis_pool_clustered(redis_dsn, cfg).await,
-            _ => crate::redis::new_redis_pool(redis_dsn, cfg).await,
+            CacheType::Redis => crate::redis::new_redis_pool(redis_dsn, cfg).await,
+            _ => panic!(
+                "This test should only be run when redis is configured as the cache provider"
+            ),
         }
     }
 
     #[tokio::test]
+    // run with `cargo test -- --ignored redis` only when redis is up and configured
+    #[ignore]
     async fn test_cache_crud_no_ttl() {
```

Comment:
Is this test captured by `cargo test -- --ignored redis` ?

Reply:
Yes. That command will run all tests whose fully qualified name contains the string `redis`. In this case the fully qualified name is: `core::cache::redis::tests::test_cache_crud_no_ttl`.